POS/Integration/06_CreateStore.md

```diff
@@ -170,26 +184,25 @@ The store wizard has been started, see [Start the store wizard](#start-the-store
 
 ![Wizard 05](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW05.png "[Store Wizard]")
 
-1. Select the warehouse you want to take the stock from in the *Warehouse* drop-down list.
+1. Click the *Warehouse* drop-down list and select the warehouse you want to take the stock from.
 
 2. Activate the *Automatically assign product to warehouse when activated in store* toggle to automate the assignment process. It is recommended to activate this option.
 
-    > [Info] A product must have stock in a warehouse in order to be sold from that warehouse. Products only have stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding channels offer is activated.    
+    > [Info] A product must have a stock level in a warehouse in order to be sold from that warehouse. Products only have stock level in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding sales channels offer is activated. 
```

Comment:
Ich würde hier bei "stock" bleiben, im Sinne von "das Produkt ist vorrätig". Vielleicht ein wenig umformulieren: A product must be in stock in a warehouse... 

stock level = physical stock in warehouse
stock value = calculated value to display in the sales channel (bedeutet aber nicht, dass man tatsächlich nicht mehr auf Vorrat hat)

Reply:
done